src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/PeopleController.kt

```diff
@@ -5,6 +5,7 @@ import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.PeopleApiDelegate
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Adjudication
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Person
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.PersonAcctAlert
```

Comment:
I'm still finding it confusing that this `PersonAcctAlert` class is a build artefact only (`build/generated/src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/api/model/PersonAcctAlert.kt`) and not part of the repo / narrative, but not sure there anything we can do about this?

Reply:
We could probably change the config so that the generated classes are output into a different location in the normal `src` directory and check them into source control.

I feel quite strongly that we shouldn't though, since generating the models from the OpenAPI spec is deterministic I'd argue there's nothing to review so it'd be adding extra LOC to pull requests for no benefit.  i.e. Since we've already reviewed the OpenAPI spec changes, we'd be reviewing the same thing twice with the caveat that we can't really change the generated code anyway.